addons/mail/static/src/core/common/persona_model.js

```diff
@@ -144,6 +146,17 @@ export class Persona extends Record {
         return this.store.DEFAULT_AVATAR;
     }
 
+    get presenceChannel() {
+        const parts = [
+            "odoo_presence",
+            `${this.type === "partner" ? "res.partner" : "mail.guest"}_${this.id}`,
+        ];
+        if (!this.store.self.isInternalUser) {
+            parts.push(this.im_status_access_token);
```

Comment:
We might want to not subscribe to users for which we didn't receive the token to avoid warnings in the python side as much as possible?